### PR TITLE
Octoicons can be easily used by including as script in an html file

### PR DESCRIPTION
**Pull request by shaminmeerankutty**
When looking through octoicons I have found that it is very hard to include octoicons to a simple html file. There is no way without a node or ruby project. So to make it simple I have added an `octoicons_web`. It does have the `octoicons.min.js` in the `dist` folder that can be included to a simple html file and use all those icons in the octoicons by just including the script and an html tag like `<div data-icon="arrow-both"></div>` Also we can specify the width and the height. Now even with a html beginner can use octoicons.

### CODE REVIEW

**Comment by shaminmeerankutty** [comment]
This fixes #216,  #217

File: lib/octoicons_web/src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "viewBox": "0 0 " + icon_data.width + " " + icon_data.width,
+    "class": "octicon octicon-" + name,
+    "aria-hidden": "true"
+  }
+  icon.insertAdjacentHTML("beforebegin", generateIcon(icon_data))
+  icon.parentNode.removeChild(icon);
+})
+
+function generateIcon(icon_data){
+  var icon =  "<svg " + htmlAttributes(icon_data, icon_data.options) + ">" + icon_data.path + "</svg>"
```
Comment (KartoffelCheetah) [inline lib/octoicons_web/src/index.js]: You are passing icon and icon.options to htmlAttributes.
It is not consistent with htmlAttributes's [definition](https://github.com/primer/octicons/pull/242/commits/fee22773e8591520c37f46922d0f7a1a081d844a#diff-64f0a99e7e5fb7797c348ba46ce3dca8R27).

Comment (shaminmeerankutty) [reply lib/octoicons_web/src/index.js]: @KartoffelCheetah Sorry my bad. I will fix it.

**Comment by aparcar** [comment]
Works here, thanks!

**Comment by cyberalien** [comment]
I offer alternative solution (opened a different issue, so I won't hijack this one): https://github.com/primer/octicons/issues/263

Advantages:
* It is automatically updated from NPM package daily, so it requires no maintenance.
* It downloads only icons that are used on page.
* It is universal solution, not designed specifically for Octicons, so it can be used to combine multiple icon sets on same page: Octicons, FontAwesome, Material Design Icons and others.
* Works with dynamic JavaScript. It observes DOM for changes and loads/renders icons whenever they are added dynamically, such as AJAX forms.

**Pull request closed without merging**